pepper-apis/ddp-common/src/main/java/org/broadinstitute/ddp/db/dto/CancerItem.java

```diff
@@ -0,0 +1,60 @@
+package org.broadinstitute.ddp.db.dto;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * The name of a cancer and its language
+ */
+public class CancerItem {
+
+    @SerializedName("name")
+    private final String cancerName;
+
+    @SerializedName("language")
+    private final String isoLanguageCode;
+
+    public CancerItem(String cancerName, String isoLanguageCode) {
+        this.cancerName = cancerName;
+        this.isoLanguageCode = isoLanguageCode;
+    }
+
+    public String getCancerName() {
+        return cancerName;
+    }
+
+    public String getIsoLanguageCode() {
+        return isoLanguageCode;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
`equals()` and `hashCode()` are here because `CancerStore` deals in `Sets`, and  although the new unique key constraint will help avoid duplicates, some tests were looking to verify that duplicates were filtered out at the code level.